i18n/de.po

```diff
@@ -18,446 +18,1140 @@ msgstr ""
+msgid "Pattern matching"
+msgstr "Muster-Erkennung"
 
-#: templates/index.html:31
-msgid "taylor (laurent) series"
-msgstr ""
+#: templates/features.html:65
+msgid "Polynomials"
+msgstr "Polynome"
 
-#: templates/index.html:32
-msgid "substitution"
-msgstr ""
+#: templates/features.html:69
+msgid " division, gcd, ... "
+msgstr " Division, GCD, ... "
 
-#: templates/index.html:32
-msgid "or"
-msgstr ""
+#: templates/features.html:70
+msgid "Factorization"
+msgstr "Faktorisierung"
 
-#: templates/index.html:34
-msgid "arbitrary precision integers, rationals and floats"
-msgstr ""
+#: templates/features.html:71
+msgid "Square-free decomposition"
+msgstr "Quadratfreie Zerlegung"
 
-#: templates/index.html:35
-msgid "noncommutative symbols"
-msgstr ""
+#: templates/features.html:72
+msgid "Gröbner bases"
+msgstr "Gröbnerbasen"
+
+#: templates/features.html:73
+msgid "Partial fraction decomposition"
+msgstr "Teilbruchzerlegung"
+
+#: templates/features.html:74
+msgid "Resultants"
+msgstr "Resultanten"
 
-#: templates/index.html:36
-msgid "pattern matching"
+#: templates/features.html:80
+msgid "Calculus"
 msgstr ""
 
-#: templates/index.html:40
-msgid "Then there are SymPy modules for these tasks:"
+#: templates/features.html:82
+msgid "Integration Documentation"
+msgstr "Integrationsdokumentation"
+
+#: templates/features.html:85
+msgid "Limits:"
+msgstr "Begrenzungen:"
+
+#: templates/features.html:89
+msgid "Differentiation"
+msgstr "Differenzierung"
+
+#: templates/features.html:92
+msgid "Integration:"
+msgstr "Integration:"
+
+#: templates/features.html:93
+msgid "It uses extended Risch-Norman heuristic"
+msgstr "Es verwendet die erweiterte Risch-Norman-Heuristik"
+
+#: templates/features.html:95
+msgid "Taylor (Laurent) series"
+msgstr "Taylor-Reihe (Laurent-Reihe)"
+
+#: templates/features.html:101
+msgid "Solving equations"
+msgstr "Lösen von Gleichungen"
+
+#: templates/features.html:105
+msgid "Polynomial equations"
+msgstr "Polynomielle Gleichungen"
+
+#: templates/features.html:106
+msgid "Algebraic equations"
+msgstr "Algebraische Gleichungen"
+
+#: templates/features.html:107
+msgid "Differential equations"
+msgstr "Differenzial Gleichungen"
+
+#: templates/features.html:108
+msgid "Difference equations"
+msgstr "Differenzielle Gleichungen"
+
+#: templates/features.html:109
+msgid "Systems of equations"
+msgstr "Gleichungssysteme"
+
+#: templates/features.html:110
+msgid "Diophantine equations:"
+msgstr "Diophantische Gleichungen:"
+
+#: templates/features.html:116
+msgid "Combinatorics"
+msgstr "Kombinatorik"
+
+#: templates/features.html:120
+msgid "Permutations"
+msgstr "Permutationen"
+
+#: templates/features.html:121
+msgid "Combinations"
+msgstr "Kombinationen"
+
+#: templates/features.html:122
+msgid "Partitions"
 msgstr ""
 
-#: templates/index.html:42
+#: templates/features.html:123
+msgid "Subsets"
+msgstr "Teilmengen"
+
+#: templates/features.html:124
+msgid "Permutation Groups"
+msgstr "Permutationengruppen"
```

Comment:
I think this should be 'Permutationsgruppen'.

Reply:
https://de.wikipedia.org/wiki/Permutationsgruppe
```suggestion
msgstr "Permutationsgruppen"
```